e2e/src/usecases/OffRamps.js

```diff
@@ -48,13 +48,6 @@ export default offRamps = () => {
       await waitForElementId('RNWebView')
       await expect(element(by.text('Bidali'))).toBeVisible()
     })
-
-    // TODO(tomm) debug why this is failing on Android
-    it(':ios: Then should not be able to spend CELO', async () => {
```

Comment:
I removed this test because it was failing and i don't think it works as intended - i was surprised to find that it's not straighforward to test if a button is disabled using detox because detox only has access to native elements (not JS ones). https://github.com/wix/Detox/issues/3203

this was previously passing because `celoRadioButton` returns a native element which doesn't have any property called `enabled`. i think that e2e tests are better for testing flows rather than display state, so i didn't feel too bad about moving this test into the FiatExchangeCurrency unit test. wdyt @MuckT ?